app/common/azure_blob_storage_handler.py

```diff
@@ -0,0 +1,50 @@
+import os
+import uuid
+
+from azure.storage.blob import BlobServiceClient, ContentSettings
+from sentry_sdk import capture_exception
+
+
+def get_container_or_create_if_not_exist(name="default"):
+    connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+    blob_service_client = BlobServiceClient.from_connection_string(connection_string)
+    container = blob_service_client.get_container_client(name)
+    if container.exists():
+        return container
+    try:
+        container = blob_service_client.create_container(name, public_access="blob")
+        return container
+    except Exception as e:
+        print(f"Error: {e}")
```

Comment:
```suggestion
        capture_exception(e)
```
Vet vi noe om hva slags unntak som heves i dette tilfellet og det under?